content/en/logs/guide/logs-rbac.md

```diff
@@ -24,10 +24,6 @@ Logs might contain **sensitive information** that could either get [scrubbed][1]
 
 This guide provides a methodology in developing customized Datadog roles that allows users to access logs and log features in a compliant manner.
 
-<div class="alert alert-warning">
-Creating and modifying custom roles is an opt-in feature. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
-</div>
-
```

Comment:
Once custom roles is GA, this will no longer be the case. There will be a way to enable custom roles directly through the UI.